### DISCUSSION TITLE
elmecdn 和 jsd 获取到的代码是不是不一样啊？

### DISCUSSION THREAD

**Discussion by wgx-dh** [YunYouJun/hexo-theme-starry#298] [Q&A 疑难杂症]
如题   是不是 有些 js 代码 
用 https://cdn.jsdelivr.net/npm 获取 
和 
用 https://npm.elemecdn.com 获取到的不一样啊

**Comment by YunYouJun** [comment]
我认为该问题与本项目无关……

我推荐你直接替换为 `https://fastly.jsdelivr.net/npm`。

elemecdn 可能没有实时更新。
